src/apps/projinfo_lib.cpp

```diff
@@ -0,0 +1,2238 @@
+            i++;
+            auxDBPath.push_back(argv[i]);
+        } else if (arg == "--guess-dialect") {
+            guessDialect = true;
+        } else if (arg == "--authority" && i + 1 < argc) {
+            i++;
+            authority = argv[i];
+            outputOpt.allowedAuthorities = split(authority, ',');
+        } else if (arg == "--identify") {
+            identify = true;
+        } else if (arg == "--show-superseded") {
+            showSuperseded = true;
+        } else if (arg == "--lax") {
+            outputOpt.strict = false;
+        } else if (arg == "--allow-ellipsoidal-height-as-vertical-crs") {
+            outputOpt.allowEllipsoidalHeightAsVerticalCRS = true;
+        } else if (arg == "--hide-ballpark") {
+            outputOpt.ballparkAllowed = false;
+        } else if (ci_equal(arg, "--3d")) {
+            promoteTo3D = true;
+        } else if (ci_equal(arg, "--normalize-axis-order")) {
+            // Undocumented for now
+            normalizeAxisOrder = true;
+        } else if (arg == "--output-id" && i + 1 < argc) {
+            i++;
+            const auto tokens = split(argv[i], ':');
+            if (tokens.size() != 2) {
+                strm.cerr << "Invalid value for option --output-id"
+                          << std::endl;
+                return usage(strm);
+            }
+            outputOpt.outputAuthName = tokens[0];
+            outputOpt.outputCode = tokens[1];
+        } else if (arg == "--dump-db-structure") {
+            dumpDbStructure = true;
+        } else if (arg == "--list-crs") {
+            listCRSSpecified = true;
+            if (i + 1 < argc && argv[i + 1][0] != '-') {
+                i++;
+                listCRSFilter = argv[i];
+            }
+        } else if (ci_equal(arg, "--searchpaths")) {
+#ifdef _WIN32
+            constexpr char delim = ';';
+#else
+            constexpr char delim = ':';
+#endif
+            const auto paths = split(proj_info().searchpath, delim);
+            for (const auto &path : paths) {
+                strm.cout << path << std::endl;
+            }
+            return 0;
+        } else if (ci_equal(arg, "--remote-data")) {
+#ifdef CURL_ENABLED
+            if (proj_context_is_network_enabled(ctx)) {
+                strm.cout << "Status: enabled" << std::endl;
+                strm.cout << "URL: " << proj_context_get_url_endpoint(ctx)
+                          << std::endl;
+            } else {
+                strm.cout << "Status: disabled" << std::endl;
+                strm.cout << "Reason: not enabled in proj.ini or "
+                             "PROJ_NETWORK=ON not specified"
+                          << std::endl;
+            }
+#else
+            strm.cout << "Status: disabled" << std::endl;
+            strm.cout << "Reason: build without Curl support" << std::endl;
+#endif
+            return 0;
+        } else if (arg == "-?" || arg == "--help") {
+            return usage(strm);
```

Comment:
The code in master is considering `--help` an error, and returns 1 (probably because it was simpler)
I kept the same behaviour in the code, but it is easy to change with this.
```suggestion
            usage(strm);
            return 0;
```
Should I do it?

Reply:
Well, it is a bit more code if we want to be consistent, but easy. I have it already done in my computer.